extension/mergeSort.js

```
@@ -0,0 +1,38 @@
+function mergeSort( initialSet = [] ){
+  if( initialSet == null ){
+    return 'Cannot sort null'
```

these should probably throw errors instead of return this error message as a string